src/components/search-browser/SearchBrowser.js

```diff
@@ -277,17 +277,15 @@ export default class SearchBrowser extends PureComponent {
     } = this.props
 
     this.setState({ search, query })
-
     if (query.trigger === SERVICES_TRIGGER) {
       onShowServices({ query, services })
       onLoadServicesStats({ search: search.replace(QUERY_REGEX, '') })
-    } else if (search) {
-      onShowResults()
-      onChange({
-        search,
-        filters: filters.map(filter => find(services, { id: filter }).key),
-      })
     }
+    onShowResults()
```

Comment:
We trigger showing the services, but showing the results as well since  there is no early return. How does this play together?